mpf/config_players/segment_display_player.py

```diff
@@ -61,9 +61,15 @@ def play(self, settings, context, calling_context, priority=0, **kwargs):
                 display.set_flashing(FlashingType.FLASH_MATCH)
             elif action == "no_flash":
                 display.set_flashing(FlashingType.NO_FLASH)
+            elif action == "set_color":
```

Comment:
I first was a bit confused here. So we always set_color. What is the purpose of the command then? Or do we want to move `display.set_color(s['color'])` into this method?

Reply:
Originally the `set_color()` method was called here, but that required *two* player calls to set both the text and the color. I imagine that most use cases will want color and text to be set at the same time, so I moved the `set_color()` out below.

But if somebody wants to change the color without affecting what's displayed, the `set_color` action will allow that.

Reply:
Maybe we could call set color on `set_color` and `add`? That would be consistent.

Reply:
I foresee a situation like this:
```
segment_display_player:
  mode_hurryup_started:
    seg_display_0:
      action: set_color
      color: orange
  hurryup_almost_finished:
      seg_display_0:
        action: flash
        color: red
  mode_hurryup_stopped:
      seg_display_0:
        text: "Time Out"
        color: white
```

My goal was to abstract the color from any other behavior, so any action the user wants to make can change the color at the same time without an extra command. 

But I've also never used a segment display before, so I'm just making things up as I go 😛